src/orquestra/sdk/secrets/_api.py

```diff
@@ -46,6 +46,8 @@ def get(
         orquestra.sdk.exceptions.UnauthorizedError: when the authorization with the
             remote vault failed.
     """
+    if _exec_ctx.global_context == _exec_ctx.ExecContext.WORKFLOW_BUILD:
+        return t.cast(str, _dsl.Secret(name=name, config_name=config_name))
```

Comment:
**issue**: returning `_dsl.Secret` "future" is inconsistent with the current docstring. I know this is what we want for the DSL purposes; please mention how the behavior of this function changes depending on the place where its used.

Reply:
wdyt? d0438b9